examples/with_great_expectations/with_great_expectations/ge_demo.py

```diff
@@ -1,9 +1,13 @@
-from dagster import job, op
+from dagster import Config, job, op
 from dagster._utils import file_relative_path
-from dagster_ge.factory import ge_data_context, ge_validation_op_factory
+from dagster_ge.factory import ge_validation_op_factory
 from pandas import read_csv
 
 
+class GEOpConfig(Config):
+    csv_path: str = file_relative_path(__file__, "./data/succeed.csv")
+
+
 @op
 def read_in_datafile(csv_path):
     return read_csv(csv_path)
```

Comment:
maybe this is a feature of pydantic config i missed, but wouldn't this need to be 
```
@op
def read_in_datafile(config: GEOpConfig):
    return read_csv(config.csv_path)
```

Reply:
oops. was just testing while i opened this pr 😓